db/migrate/20151101142959_collaborations.rb

```diff
@@ -0,0 +1,12 @@
+class Collaborations < ActiveRecord::Migration
+  def change
+    create_table :collaborations do |t|
+      t.integer :user_id
+      t.integer :wiki_id
+      t.timestamps
+    end
+    add_index :collaborations, :id, unique: true
```

Comment:
No need to add an index on the primary key (`id`) column. `create_table ...` adds the primary key index by default.